srcpkgs/reaper/files/application-x-reaper.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you forgot to install this, tho I kinda hate the idea of carrying it :/

Reply:
Oh, whoops, no I forgot to remove it; this is installed by the install script but is unnessecary, update-mime-database generates basically this.